unittests/AST/ASTImporterTest.cpp

```diff
@@ -461,6 +479,13 @@ class ASTImporterTestBase : public ParameterizedTestsFixture {
   }
 };
 
+class ASTImporterLangParamTestBase
```

Comment:
Perhaps this should be part of `ASTImporterTestBase`.  I don't see too much benefits for having a separate class for this. `TestImportBase` is also derived from `::testing::WithParamInterface<ArgVector>`, to be consistent, I think `ASTImporterTestBase` should derive from that too.

Reply:
We can not add the test parameter type here if it should be possible to have tests with different kinds of parameters (and the same base class for the fixture). The test can have only one parameter type and no derived test can extend the supported parameter types.

Reply:
OK.